api/src/db/DynamoBusinessesDataClient.test.ts

```diff
@@ -39,6 +39,7 @@ describe("DynamoBusinessesDataClient", () => {
     taxFilingData: generateTaxFilingData({
       filings: [],
     }),
+    version: 1,
```

Comment:
pebble - should this be the same version as userData?

Reply:
Yh we discussed yesterday, This is going to have the same version as the userData.